trunk/cli/src/commands/install.rs

```diff
@@ -137,17 +127,15 @@ impl SubCommand for InstallCommand {
             tempfile.write_reader(gz)?;
             tempfile.rewind()?;
             let input = tempfile;
-            install(input, extension_dir, package_lib_dir, bitcode_dir, sharedir).await?;
+            install(input, package_lib_dir, sharedir).await?;
```

Comment:
installation directories are being simplified to just sharedir or pkglibdir instead of each file type. File type still used to determine if it is under sharedir or pkglibdir

![image](https://user-images.githubusercontent.com/7516283/234673661-6408591f-d865-44c3-babe-d6ccf6f7f17f.png)